google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStub.java

```diff
@@ -874,6 +887,110 @@ public Map<String, String> extract(
     return traced.withDefaultCallContext(clientContext.getDefaultCallContext());
   }
 
+  /**
+   * Creates a callable chain to handle streaming ReadChangeStream RPCs. The chain will:
+   *
+   * <ul>
+   *   <li>Convert a {@link ReadChangeStreamQuery} into a {@link ReadChangeStreamRequest} and
+   *       dispatch the RPC.
+   *   <li>Upon receiving the response stream, it will produce a stream of ChangeStreamRecordT. In
+   *       case of mutations, it will merge the {@link ReadChangeStreamResponse.DataChange}s into
+   *       logical mutations. The actual change stream record implementation can be configured by
+   *       the {@code changeStreamRecordAdapter} parameter.
+   *   <li>TODO: Retry/resume on failure.
+   *   <li>Add tracing & metrics.
+   * </ul>
+   */
+  public <ChangeStreamRecordT>
+      ServerStreamingCallable<ReadChangeStreamQuery, ChangeStreamRecordT>
+          createReadChangeStreamCallable(
+              ChangeStreamRecordAdapter<ChangeStreamRecordT> changeStreamRecordAdapter) {
+    ServerStreamingCallable<ReadChangeStreamRequest, ChangeStreamRecordT> readChangeStreamCallable =
+        createReadChangeStreamBaseCallable(
```

Comment:
Do we need to create a base callable for this? ReadRows request has a base callable because the callable is used in multiple places (ReadRow callable, ReadRowsCallable and bulkReadRows callable). Maybe you could merge the 2 methods into one.

Reply:
I see! Yeah we don't need two methods. Merged them into one.